package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=7.14.0"
   },
   "devDependencies": {
+    "at-least-node": "1.0.0",
```

Comment:
__NOTE__ i needed to install this otherwise the javascript part of the build process would fail